src/tools/fuzzing.h

```diff
@@ -77,6 +81,8 @@ class TranslateToFuzzReader {
   Module& wasm;
 
 private:
+  // Whether the module will be tested in a closed-world environment.
+  bool closedWorld;
```

Comment:
This isn't actually used in this PR, so should we move it to a later PR that uses it?

Reply:
I put it here because it receives the info that is sent in the other half of the PR, so it shows both parts.

And the other half of the PR isn't "used" either, it has no effect :smile: 

But happy to do it either way, what do you prefer?

Reply:
Good point, I was missing the line in `wasm-opt.cpp` to send it. I pulled it from the next PR to here now.